tests/components/openweathermap/test_config_flow.py

```diff
@@ -155,10 +159,14 @@ async def test_form_options(hass):
 
 async def test_form_invalid_api_key(hass):
     """Test that the form is served with no input."""
-    mocked_owm = _create_mocked_owm(True)
+    mocked_owm = _create_mocked_owm()
 
     with patch(
-        "pyowm.weatherapi25.owm25.OWM25",
+        "homeassistant.components.openweathermap.config_flow.OWM",
+        return_value=mocked_owm,
```

Comment:
If we set a side effect we don't need to set return value.

Reply:
Done.